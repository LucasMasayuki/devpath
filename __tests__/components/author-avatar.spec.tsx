import React from 'react'
import renderer from 'react-test-renderer'
import Author from '@/types/author'
import AuthorAvatar from '@/components/author-avatar'

describe('AuthorAvatar', () => {
  it('should render AuthorAvatar', () => {
    const author: Author = {
      name: 'test',
      picture: 'test',
    }

    const component = renderer.create(<AuthorAvatar author={author} />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
