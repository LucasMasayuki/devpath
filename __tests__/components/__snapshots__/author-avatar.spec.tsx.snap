// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthorAvatar should render AuthorAvatar 1`] = `
<div
  className="css-1yh2cij"
>
  <span
    className="chakra-avatar css-1ruhx4w"
  >
    <div
      aria-label="test"
      className="chakra-avatar__initials css-0"
      role="img"
    >
      t
    </div>
  </span>
  <span
    className="css-xf6agq"
  >
    test
  </span>
</div>
`;
